python/lsst/ip/diffim/utils.py

```diff
@@ -1080,20 +1080,31 @@ def detectDipoleSources(self, doMerge=True, diffim=None, detectSigma=5.5, grow=3
             return detectTask, schema
 
 
-def getPsfFwhm(psf):
-    """Calculate the FWHM in pixels of a supplied PSF.
+def getPsfFwhm(psf, average=True):
+    """Directly calculate the horizontal and vertical widths
+    of a PSF at half its maximum value.
 
     Parameters
     ----------
     psf : `lsst.afw.detection.Psf`
         Point spread function (PSF) to evaluate.
+    average : `bool`, optional
+        Set to return the average width.
 
     Returns
     -------
     psfSize : `float`
```

Comment:
Does the returns description need to be changed since you could also be returning what appears to be a tuple when average = False?